xpk.py

```diff
@@ -999,6 +1007,59 @@ def run_gke_cluster_create_command(args) -> int:
   return 0
 
 
+def create_cluster_configmap(args):
+  """Run the Create GKE Cluster ConfigMap request.
+
+  Args:
+    args: user provided arguments for running the command.
+
+  Returns:
+    0 if successful and 1 otherwise.
+  """
+  system = UserFacingNameToSystemCharacteristics[args.tpu_type]
+  data = f'{args.tpu_type}: "{int(args.num_slices) * system.vms_per_slice}"'
```

Comment:
Just to note that @michelle-yooh is adding gpu support which will need to modify this a bit. https://github.com/google/xpk/pull/49. So we just have to be aware depending on the order that things get merged!

Reply:
Thanks for the heads up.